src/components/DisplayNames/DisplayNamesWithTooltip.tsx

```diff
@@ -46,13 +46,14 @@ function DisplayNamesWithToolTip(props) {
     return (
         // Tokenization of string only support prop numberOfLines on Web
         <Text
-            style={[...props.textStyles, styles.pRelative, props.numberOfLines === 1 ? styles.noWrap : {}]}
-            numberOfLines={props.numberOfLines || undefined}
-            ref={(el) => (containerRef.current = el)}
+            style={[textStyles, styles.pRelative, numberOfLines === 1 ? styles.noWrap : {}]}
+            numberOfLines={numberOfLines || undefined}
+            ref={containerRef}
```

Comment:
why this logic change?

Reply:
@kubabutkiewicz There are two reasons:
1) With old logic it gives ts error: `Cannot assign to  current  because it is a read-only property.`
2) I console log `containerRef` with old and updated logic and it seems to have the same result.